src/main/groovy/org/labkey/gradle/plugin/extension/ModuleExtension.groovy

```diff
@@ -81,25 +79,30 @@ class ModuleExtension
         if (propertiesFile.exists()) {
             Properties props = new Properties()
             PropertiesUtils.readProperties(propertiesFile, props)
+
             this.modProperties.putAll(props)
             if (logDeprecations) {
                 List<String> deprecationMsgs = []
-                if (this.modProperties.get(MODULE_DEPENDENCIES_PROPERTY))
-                    deprecationMsgs += "The '" + MODULE_DEPENDENCIES_PROPERTY + "' property is no longer supported as of gradlePlugin version 1.25.0 (LabKey Server version 21.3.0)." +
-                            " Declare the dependency in the module's build.gradle file instead using the 'modules' configuration." +
-                            " See https://www.labkey.org/Documentation/wiki-page.view?name=gradleDepend for more information."
-
-                if (this.modProperties.get("ConsolidateScripts"))
-                    deprecationMsgs += "The 'ConsolidateScripts' property is no longer supported."
-                if (this.modProperties.get("Version"))
-                    deprecationMsgs += "The 'Version' property is no longer supported."
+//          Follow this pattern to deprecate a property in module.properties
+//                if (this.modProperties.get("OldProperty"))
+//                    deprecationMsgs += "The OldProperty property is no longer supported."
                 if (!deprecationMsgs.isEmpty())
                     project.logger.quiet("${propertiesFile.absolutePath}: Deprecated or unsupported properties detected.\n\t"
-                            + deprecationMsgs.join("\n\t")
-                            + "\nRefer to https://www.labkey.org/Documentation/wiki-page.view?name=includeModulePropertiesFile for the current set of supported properties.")
+                        + deprecationMsgs.join("\n\t")
+                        + "\nRefer to https://www.labkey.org/Documentation/wiki-page.view?name=includeModulePropertiesFile for the current set of supported properties.")
             }
-            if (this.modProperties.get(MODULE_DEPENDENCIES_PROPERTY))
-                this.modProperties.remove(MODULE_DEPENDENCIES_PROPERTY)
+
+            List<String> unsupportedMsgs = []
```

Comment:
Can we maybe leave a note here about when we can remove these checks altogether? Seems like probably the next time we change the version of LabKey we're compatible with would be sufficient.

Reply:
Good idea. It's difficult to decide on timeframe because we don't know who is affected (if anyone), how often they build their modules, and what release they're building against (they'll see the warnings & exceptions only if they're building on a branch with the new plugins version). My compromise proposal: we deprecate for one year and throw for one year after that. LMK if you disagree.

And thanks for reminding me: since I'm switching the default value of `ManageVersion`, the next plugins version will require 25.2+.

Reply:
Your compromise proposal works for me.